src/components/configurator/Features.js

```diff
@@ -293,25 +299,30 @@ function enforceEitherReactOrVueOrSvelte(
       ...allFeatureStates,
       React: !setToSelected,
       Svelte: !setToSelected,
+      'No library': !setToSelected,
     };
     // deselect vue if user selects react
-  }
-  if (affectedFeature === 'React') {
-    if (setToSelected) {
-      return {
-        ...allFeatureStates,
-        Vue: !setToSelected,
-        Svelte: !setToSelected,
-      };
-    }
-  } else if (affectedFeature === 'Svelte') {
-    if (setToSelected) {
-      return {
-        ...allFeatureStates,
-        Vue: !setToSelected,
-        React: !setToSelected,
-      };
-    }
+  } else if (affectedFeature === 'React' && setToSelected) {
```

Comment:
maybe rename function to `enforceEitherReactOrVueOrSvelteOrNoLibrary` . hmm...starts to get long :D

Reply:
what about `enforceMainLibrary`?

Reply:
yes, that's better!